synapse/lib/stormlib/macro.py

```diff
@@ -148,6 +144,13 @@ class LibMacro(s_stormtypes.Lib):
         {'name': 'list', 'desc': 'Get a list of Storm Macros in the Cortex.',
          'type': {'type': 'function', '_funcname': '_funcMacroList',
                   'returns': {'type': 'list', 'desc': 'A list of ``dict`` objects containing Macro definitions.', }}},
+        {'name': 'mod', 'desc': 'Modify user editable properties of a Storm Macro.',
+         'type': {'type': 'function', '_funcname': '_funcMacroMod',
+                  'args': (
+                      {'name': 'name', 'type': 'str', 'desc': 'Name of the Storm Macro to add or modify.', },
```

Comment:
is this intended to be able to add as well as modify?

Reply:
Nope, good catch, thanks!